python/doc/examples/calibration/least_squares_and_gaussian_calibration/plot_calibration_flooding.py

```diff
@@ -332,209 +350,211 @@ def functionFlooding(X):
 # We can compute a 95% confidence interval of the parameter :math:`\theta^\star`.
 #
 # This confidence interval is based on bootstrap, based on a sample size equal
-# to 100 (as long as the value of the `ResourceMap` key "NonLinearLeastSquaresCalibration-BootstrapSize" is unchanged).
-# This confidence interval reflects the sensitivity of the optimum to the variability in the observations.
+# to 100 (as long as the value of the `ResourceMap` key
+# "NonLinearLeastSquaresCalibration-BootstrapSize" is unchanged).
+# This confidence interval reflects the sensitivity of the optimum
+# to the variability in the observations.
 
 # %%
 thetaPosterior = calibrationResult.getParameterPosterior()
 print(thetaPosterior.computeBilateralConfidenceIntervalWithMarginalProbability(0.95)[0])
 
 # %%
-# In this case, the value of the parameter :math:`K_s` is quite accurately computed, but there is a relatively large uncertainty on the values of :math:`Z_v` and :math:`Z_m`.
+# In this case, the values of the parameters are quite accurately
+# computed.
 
 # %%
 graph = calibrationResult.drawObservationsVsInputs()
 graph.setLegendPosition("topleft")
 view = viewer.View(graph)
 
 # %%
-# We see that there is a good fit after calibration, since the predictions after calibration (i.e. the green crosses) are close to the observations (i.e. the blue crosses).
+# We see that there is a good fit after calibration, since the predictions after calibration
+# are close to the observations.
 
 # %%
 graph = calibrationResult.drawObservationsVsPredictions()
 view = viewer.View(graph)
 
 # %%
-# We see that there is a much better fit after calibration, since the predictions are close to the diagonal of the graphics.
+# We see that there is a much better fit after calibration, since the
+# predictions are close to the diagonal of the graphics.
 
 # %%
 observationError = calibrationResult.getObservationsError()
 print(observationError)
 
 # %%
-# We can see that the observation error is close to have a zero mean and a standard deviation equal to 0.1.
+# We can see that the observation error has a sample mean close to zero and a
+# sample standard deviation approximately equal to 0.11.
 
 # %%
```

Comment:
```suggestion
```
To make sphinx_gallery_thumbnail_number = 5 appear as code rather than RST.

Reply:
Ok. Fixed.